contracts/test/stateverifier.test.ts

```diff
@@ -84,27 +102,27 @@ describe("Verify", function () {
         pedersenHash.address
       );
 
+      //////////StarknetCoreContract STUB DEPLOYMENT(ONLY FOR TESTING)
       const StarknetCoreContractStub = await ethers.getContractFactory(
         "StarknetCoreContractStub"
       );
       starknetCoreContractStub = await StarknetCoreContractStub.deploy();
       await starknetCoreContractStub.deployed();
-
       console.log(
         "StarknetCoreContractStub contract has been deployed to: ",
         starknetCoreContractStub.address
       );
 
+      //////////VERIFIER + L1 RESOLVER DEPLOYMENT (they are the same class)
       const SNResolverStub = await ethers.getContractFactory(
         "SNResolverStub"
       );
-
       const proofProxy = await upgrades.deployProxy(
         SNResolverStub,
-        [pedersenHash.address, starknetCoreContractStub.address, ["https://localhost:9545/{sender}/{data}.json"], '0x7412b9155cdb517c5d24e1c80f4af96f31f221151aab9a9a1b67f380a349ea3'],
+        [pedersenHash.address, poseidon3.address, starknetCoreContractStub.address, ["https://localhost:9545/{sender}/{data}.json"], '0x7412b9155cdb517c5d24e1c80f4af96f31f221151aab9a9a1b67f380a349ea3'],
```

Comment:
same here, we can use `L2_RESOLVER_ADDRESS`

Reply:
see comment above about L2_RESOLVER_ADDRESS